themes/marquee/layout/_partial/footer.ejs

```diff
@@ -3,7 +3,7 @@
     <div class="row">
       <div class="col-md-8 offset-md-2">
         <h2>Code of Conduct</h2>
-        CampJS takes pride in being an inclusive and welcoming event for people from all walks of life.
+        CampJS ios committed to creating an inclusive and welcoming event for people from all walks of life.
```

Comment:
typo

Reply:
Thanks for the spot (and obvs Androids are welcome too)